pkg/loop/internal/pb/keystore/keystore.proto

```diff
@@ -0,0 +1,136 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/keystore;keystorepb";
+
+package loop.internal.pb.keystore;
+
+service Keystore {
+  rpc Sign(SignRequest) returns (SignResponse);
+  rpc SignBatch(SignBatchRequest)returns (SignBatchResponse);
+  rpc Verify(VerifyRequest)returns (VerifyResponse);
+  rpc VerifyBatch(VerifyBatchRequest)returns (VerifyBatchResponse);
+
+  rpc List(ListRequest)returns (ListResponse);
+
+  rpc RunUDF(RunUDFRequest)returns (RunUDFResponse);
+
+  //
```

Comment:
```suggestion
```

Reply:
Removed in [`9bc2713` (#837)](https://github.com/smartcontractkit/chainlink-common/pull/837/commits/9bc2713bc46d6518a758830828454e7077039dbe)